README.md

```diff
@@ -1,59 +1,61 @@
-# teracy-dev-core
+# Teracy-dev-core
 
-The core extension for teracy-dev which support all options from https://www.vagrantup.com/docs/
+The core extension for [teracy-dev](https://github.com/teracyhq/dev/tree/develop) which support all options from [vagrant](https://www.vagrantup.com/docs/)
 for teracy-dev yaml config format.
 
+## Prerequisites
```

Comment:
specify the required teracy-dev version from https://github.com/teracyhq-incubator/teracy-dev-core/blob/develop/manifest.yaml